### PR TITLE
Support for C++/C files and headers

### PR DESCRIPTION
**Pull request by srijanshetty**
Nothing major, C/C++ are practically same as JS in regard of comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,20 +23,20 @@ to extract your todos from comments.
...
-| Handlebars   | `.hbs`          | using regex. Supports `{{! }}` and `{{!-- --}}` |
-| Jade         | `.jade`         | using regex.                                    |
-| Javascript   | `.js`           | using regex. Supports `// and /* */` comments   |
-| Jsx          | `.jsx`          | using regex. Supports `// and /* */` comments.  |
-| Less         | `.less`         | using regex. Supports `// and /* */` comments.  |
-| Ruby         | `.rb`           | using regex. Supports `#` comments.             |
-| Sass         | `.sass` `.scss` | using regex. Supports `// and /* */` comments.  |
-| Stylus       | `.styl`         | using regex. Supports `// and /* */` comments.  |
-| Twig         | `.twig`         | using regex. Supports `{#  #}` and `<!-- -->`   |
-| Typescript   | `.ts`           | using regex. Supports `// and /* */` comments.  |
```
Comment (DanielOaks) [inline README.md]: Was Typescript meant to be removed?

**Comment by srijanshetty** [comment]
@DanielOaks : No, I'll get that fixed.

**Comment by srijanshetty** [comment]
@DanielOaks : Fixed

**Comment by pgilad** [comment]
Thank you! :+1:

**Pull request merged**